documentation/man/features/build.rst

```diff
@@ -51,6 +54,18 @@ OPTIONS
   This option allows the user to enable ``ccache`` usage during compilation
   tasks, which should improve compile times in subsequent compilations. 
 
+-w, \--warnings=(1 | 2 | 3 | 12 | 13 | 23 | 123):
+  This can be used to enable compilation warnings accordingly. Please check the
+  kernel's ``make help`` for more info.
```

Comment:
I think it will be more friendly if you explain the warning level 1 and 2... Actually, 1 and 2 are hard to understand; maybe we could have a string alias that is easy to understand.

Reply:
Agree!